python/tvm/relay/frontend/pytorch.py

```diff
@@ -0,0 +1,1138 @@
+                node_assign = (node_str.split(' = ')[0]).split(' : ')
+                node_name = (node_assign[0])[1:]
+                node_getattr_name = ((node_str.split(' = ')[1]).split('"')[1::2])[0]
+                node_arg = (((node_str.split(' = '))[1]).split('(')[1])[1:-2]
+
+                if node_arg in input_names:
+                    node_weight_map[node_name] = node_getattr_name
+                else:
+                    previous_map = node_weight_map[node_arg[:]]
+                    node_weight_map[node_name] = previous_map+"."+node_getattr_name
+
+                if node_getattr_name in param_names:
+
+                    value = state_dict[node_weight_map[node_name]]
+                    tensor = tvm.nd.array(value.cpu().numpy())
+                    shape = tensor.shape
+                    self._param_tensors[node_name] = tensor
+
+                    self._params[node_name] = _expr.var(node_name,
+                                                        shape=shape,
+                                                        dtype=_convert_data_type(str(value.dtype)))
+
+                    self._fn_param.append(_expr.var(node_name,
+                                                    shape=shape,
+                                                    dtype=_convert_data_type(str(value.dtype))))
+
+
+    def _parse_ops(self):
+        """ Iterate through nodes and decorate graph with constants, operators,
+        and the inputs to each operator. """
+        # Traverse nodes and add to graph
+        for node in self._trace.graph.nodes():
+
+            node_str = str(node)
+            node_assign = (node_str.split(' = ')[0]).split(' : ')
+            node_name = (node_assign[0])[1:]
+            node_expr = (node_str.split(' = ')[1]).split(',')[0]
+
+            if node.kind() == "prim::Constant":
+                node_value = '0'
+                if "None" not in node_str and node_expr != "prim::Constant()" and \
+                        "?" not in node_str:
+                    node_value = ((node_str.split(' = ')[1]).split('value=')[1]).split(']')[0]
+                self._consts[node_name] = node_value
+            elif node.kind() == "prim::ListConstruct":
+                list_shape = []
+                for input_node in node.inputs():
+                    if input_node.debugName() in self._inputs_r.keys():
+                        list_shape.append(int(self._inputs_r[input_node.debugName()]))
+                    elif input_node.debugName() in self._consts.keys():
+                        list_shape.append(int(self._consts[input_node.debugName()]))
+                    else:
+                        pass
+                self._inputs_r[node_name] = _expr.var(node_name, shape=list_shape)
+            elif node.kind() == "prim::GetAttr":
+                continue
+
+            self._add_op(node_name, node.kind(), node)
+
+    # Graph Helper Functions
+
+    def _add_op(self, op_name, operator, op_node):
```

Comment:
you don't need "operator" as dict key and it doesn't seem to be used anywhere else, so remove it from the argument